examples/components/CCFRAUD/traininsilo/models.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, input_dim) -> None:
         self.lstm = nn.LSTM(
             input_size=input_dim,
             hidden_size=128,
-            num_layers=4,
+            num_layers=1,
```

Comment:
NIT: What is the gist behind changing number of layers to 1?

Reply:
It's for the overall performance, the same as mentioned above.